src/gui/wizard/owncloudadvancedsetuppage.cpp

```diff
@@ -326,17 +326,28 @@ void OwncloudAdvancedSetupPage::slotSelectiveSyncClicked()
                 _ui.lSelectiveSyncSizeLabel->setText(QString());
             }
         } else {
-            _ui.rSyncEverything->setChecked(true);
+            setRadioChecked(_ui.rSyncEverything);
             _ui.lSelectiveSyncSizeLabel->setText(QString());
         }
         wizard()->setProperty("blacklist", _selectiveSyncBlacklist);
     }
 }
 
+void OwncloudAdvancedSetupPage::slotPlaceholderSyncClicked()
+{
+    if (!OwncloudWizard::askExperimentalPlaceholderFeature())
+        return;
+
+    _ui.lSelectiveSyncSizeLabel->setText(QString());
+    _selectiveSyncBlacklist.clear();
+
+    setRadioChecked(_ui.rPlaceholderSync);
+}
```

Comment:
It is missing the code in OwncloudSetupWizard::slotAssistantFinished  to make the placeholder active